contracts/exchange-libs/contracts/src/LibEIP712ExchangeDomain.sol

```diff
@@ -29,9 +29,11 @@ contract LibEIP712ExchangeDomain {
     // EIP712 Exchange Domain Version value
     string constant internal _EIP712_EXCHANGE_DOMAIN_VERSION = "3.0.0";
 
-    // Hash of the EIP712 Domain Separator data
-    // solhint-disable-next-line var-name-mixedcase
+    // solhint-disable var-name-mixedcase
+    /// @dev Hash of the EIP712 Domain Separator data
+    /// @return 0 Domain hash.
```

Comment:
Ah, so this is pulled in for the public function for this variable?

Reply:
Yup.